docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
+[Brigade Helm Chart][Helm chart values]. The default access mode for the
+backing PersistentVolume is `ReadWriteOnce`. This mode, although configurable,
+should not need changing.
 
-To set these at the Brigade server level, set the values below accordingly in the Brigade
-chart's `values.yaml` file:
+There are, however, other persistence options that may be useful to customize,
+such as volume size and storage class type. All configuration options can be
+seen under the `artemis.persistence` section of the Brigade chart.
 
-values.yaml
-```yaml
-worker:
-  defaultBuildStorageClass: nfs
-  defaultCacheStorageClass: nfs
-```
+As the access mode is `ReadWriteOnce`, nearly all storage class types should
+function correctly for this PersistentVolume. By default, no storage class is
+specified in the chart, which means the default storage class on the substrate
+will be employed.
 
-Then:
+[Design]: /topics/design
+[ActiveMQ Artemis]: https://activemq.apache.org/components/artemis/
 
-```console
-$ helm upgrade brigade brigade/brigade -f values.yaml
-```
+## MongoDB storage
 
-To set these at the Brigade project level, set the values below accordingly in the Brigade
-Project chart's `values.yaml` file:
+Brigade uses [MongoDB] as its backing data store. For more details on its
+function in Brigade, see the [Design] doc.
 
-values.yaml
-```yaml
-kubernetes:
-  buildStorageClass: nfs
-  cacheStorageClass: nfs
-```
+Data should be persisted even when/if MongoDB itself goes down or is restarted.
+Therefore, by default, persistence is enabled via the appropriate configuration
+on the [Brigade Helm Chart][Helm chart values]. The default access mode for the
+backing PersistentVolume is `ReadWriteOnce`. This mode, although configurable,
+should not need changing.
 
-Then:
+There are, however, other persistence options that may be useful to customize,
+such as volume size and storage class type. All configuration options can be
+seen under the `mongodb.persistence` section of the Brigade chart.
 
-```console
-$ helm upgrade my-project brigade/brigade-broject -f values.yaml
-```
+As the access mode is `ReadWriteOnce`, nearly all storage class types should
+function correctly for this PersistentVolume. By default, no storage class is
+specified in the chart, which means the default storage class on the substrate
+will be employed.
 
-Note: The project-level settings can also be configured during the "Advanced" set up if creating via the `brig` CLI:
+[MongoDB]: https://www.mongodb.com/
 
-```console
- $ brig project create
-...
-
-? Build storage class nfs
-? Job cache storage class  [Use arrows to move, type to filter, ? for more help]
-  azurefile
-  default
-  managed-premium
-❯ nfs
-  Leave undefined
-```
+## Examples
 
-If you would prefer to use the NFS provisioner as a cluster-wide default volume provider
-(and have Brigade automatically use it), you can do so by making it the default
-storage class:
+### Using an NFS Server
```

Comment:
The placement of this section threw me off. Would it make more sense if it were moved up to where shared workspace storage is discussed?

Reply:
Good call, I moved it up and reworded things... lmk what you think: https://github.com/brigadecore/brigade/pull/1586/commits/42a85411f712ebf74b8a76d27e25be9f6c45391a